src/patternfly/components/Toolbar/examples/Toolbar.md

```diff
@@ -656,6 +656,8 @@ As the toolbar component is a hybrid layout and component, some of its elements
 | `.pf-m-full-height` | `.pf-v5-c-toolbar`, `.pf-v5-c-toolbar__content-section`, `.pf-v5-c-toolbar__group` | Modifies toolbar component to full height of its container and removes vertical padding. |
 | `.pf-m-static` | `.pf-v5-c-toolbar` | Modifies expandable content section to position itself to the nearest absolutely positioned parent outside of the toolbar component. This is used primarily for masthead toolbar. |
 | `.pf-m-expanded` | `.pf-v5-c-toolbar__expandable-content` | Modifies expandable content section for the expanded state. |
+| `.pf-m-label` | `.pf-v5-c-toolbar__item` | Initiates label vertical positioning. |
+| `.pf-m-form-element` | `.pf-v5-c-toolbar__item` | Initiates form element vertical positioning. |
```

Comment:
Stupid nit, but can you change "initiates" to "Modifies for" or whatever makes sense grammatically? (maybe the ones below too?)